api/src/main/java/org/pmiops/workbench/auth/ServiceAccounts.java

```diff
@@ -1,52 +1,26 @@
 package org.pmiops.workbench.auth;
 
-import static com.google.api.client.googleapis.util.Utils.getDefaultJsonFactory;
-
-import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
-import com.google.api.client.http.HttpTransport;
-import com.google.appengine.api.appidentity.AppIdentityService;
-import com.google.appengine.api.appidentity.AppIdentityServiceFactory;
+import com.google.auth.oauth2.GoogleCredentials;
 import java.io.IOException;
 import java.util.List;
-import org.pmiops.workbench.config.WorkbenchEnvironment;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
 
 /**
  * Handles functionality related to loading service account credentials and generating derived /
  * impersonated credentials.
  */
-@Component
 public class ServiceAccounts {
 
-  private final HttpTransport httpTransport;
-
-  @Autowired
-  public ServiceAccounts(HttpTransport httpTransport) {
-    this.httpTransport = httpTransport;
-  }
-
-  public GoogleCredential.Builder getCredentialBuilder() {
-    return new GoogleCredential.Builder();
-  }
+  private static final String SIGN_JWT_URL_FORMAT =
+      "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/%s:signJwt";
 
   /**
    * Retrieves an access token for the Workbench server service account. This should be used
    * carefully, as this account is generally more privileged than an end user researcher account.
    */
-  public String workbenchAccessToken(WorkbenchEnvironment workbenchEnvironment, List<String> scopes)
-      throws IOException {
-    // When running locally, we get application default credentials in a different way than
-    // when running in Cloud.
-    if (workbenchEnvironment.isDevelopment()) {
-      GoogleCredential credential = GoogleCredential.getApplicationDefault().createScoped(scopes);
-      credential.refreshToken();
-      return credential.getAccessToken();
-    }
-    AppIdentityService appIdentity = AppIdentityServiceFactory.getAppIdentityService();
-    final AppIdentityService.GetAccessTokenResult accessTokenResult =
-        appIdentity.getAccessToken(scopes);
-    return accessTokenResult.getAccessToken();
+  public static String workbenchAccessToken(List<String> scopes) throws IOException {
+    GoogleCredentials scopedCreds = GoogleCredentials.getApplicationDefault().createScoped(scopes);
```

Comment:
Per Jay's comment above, in this case I would either change this function to accept a GoogleCredentials as a parameter (the client would pass ADC), or revert the static change and go back to a component/service